src/App.js

```diff
@@ -2,12 +2,21 @@
 import React from 'react';
 import Calculator from './components/Calculator';
 import './App.css';
+import { Routes, Route } from 'react-router-dom';
+import Home from './components/Home';
+import Navigation from './components/Navigation';
+import Quote from './components/Quote';
 
 class App extends React.Component {
   render() {
     return (
       <div className="App">
-        <Calculator />
+        <Navigation />
+        <Routes>
+          <Route path="/home" element={<Home />} />
```

Comment:
- [ OPTIONAL ] This is your main/home route so the path should be equal to just `/`.

Reply:
Thank you for this @GzimAsani!